### DISCUSSION TITLE
iOS端，加载网络资源，使用setPathAsync方法，PAGView能加载出来，PAGImageView加载不出来，请问是怎么回事？

### DISCUSSION THREAD

**Discussion by li-qingnan** [Tencent/libpag#1704] [Q&A]
iOS端，加载网络资源，使用setPathAsync方法，PAGView能加载出来，PAGImageView加载不出来，请问是怎么回事？
使用的pod方式导入的，版本是4.3.7。

**Comment by domchen** [comment]
这听起来是非常好复现的一个问题，把具体的demo上传下，避免提一句话问题，这样我们也帮不到你。
